### PR TITLE
feat(spare): add spare management for telephony and xdsl

### PR DESCRIPTION
**Pull request by Steffy29**
Ref: UXCT-177

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build#1111.0 ✘  
* 
  * build ✔ 
  * test ✘

**Comment by ovh-cds** [comment]
CDS Report build#1112.0 ✘  
* 
  * build ✔ 
  * test ✘

File: src/api/xdsl/spare/xdsl-spare.service.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+angular.module("ovh-api-services").service("OvhApiXdslSpare", function ($injector, $cacheFactory) {
+    "use strict";
+
+    var cache = $cacheFactory("OvhApiXdslSpare");
```
Comment (JeremyDec) [inline src/api/xdsl/spare/xdsl-spare.service.js]: Same, it won't be useful

**Comment by ovh-cds** [comment]
CDS Report build#1118.0 ✘  
* 
  * build ✔ 
  * test ✘

**Pull request merged**